themes/tiddlywiki/vanilla/base.tid

```diff
@@ -1792,6 +1791,12 @@ html body.tc-body.tc-single-tiddler-window {
 	width: 100%;
 }
 
+.tc-controlpanel svg.tc-image-button{
+	width: 1em;
+	height: 1em;
+	vertical-align:middle;
```

Comment:
@pmario, ah, I guess that slipped in when I was trying to figure out the right spot to put it but then didn't fully revert to the initial state

will fix this eventing